docs/source/index.rst

```diff
@@ -22,8 +22,8 @@ Head over to our :doc:`/examples/guides/quickstart_in_5_minutes` tutorial,
 and click on  |binder badge|  or on  |colab badge|  to launch it and see it in action,
 or see our :doc:`/getting-started/index` to install it locally and continue from there.
 
-.. note:: The package's output is suited for running in a jupyter environment.
-          HTML and PDF reports for the graphs may be added in the near future.
+.. note:: The package's output is suited for running in any Python environment.
+          Best viewed in Jupyer Notebooks but can be run in any ide and be saved as HTML.
```

Comment:
```suggestion
          The output is est viewed in Jupyter Notebooks (or JupyterLab), but it can also run in any ide and be exported as an HTML report.
```